lms/static/sass/base/_variables.scss

```diff
@@ -306,6 +306,10 @@ $credit-color-base: rgb(244,195,0); // accessible with black text
 // edx-specific: Studio/Staff actions
 $staff-color: $pink;
 
+// from the edX Pattern Library
```

Comment:
Forgive my ignorance, but why can't we use these directly from the pattern library?